extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputFormat.java

```diff
@@ -59,6 +70,17 @@ public boolean isSplittable()
   @Override
   public InputEntityReader createReader(InputRowSchema inputRowSchema, InputEntity source, File temporaryDirectory)
   {
+    KafkaRecordEntity kafkaInputEntity = (KafkaRecordEntity) source;
+    if (Arrays.asList(kafkaInputEntity.getRecord().headers().toArray()).contains(HEADER_V1)) {
```

Comment:
you probably want to avoid getting all headers and instead call `headers("v")` and then take the first one (if there is one), and compare the bytes directly. This wil avoid the unnecessary toArray() and Arrays.asList() calls